lib/module.js

```diff
@@ -62,12 +62,16 @@ const debug = Module._debug;
 // check if the directory is a package.json dir
 const packageMainCache = {};
 
+function getPackageJsonPath(requestPath) {
```

Comment:
You can just revert this change after you follow my suggestion in the comment below